ast/parser_test.go

```diff
@@ -5053,6 +5103,60 @@ func assertParseModule(t *testing.T, msg string, input string, correct *Module,
 
 }
 
+func assertParseModuleJSONOptions(t *testing.T, msg string, input string, opts ...ParserOptions) {
+	opt := ParserOptions{}
+	if len(opts) == 1 {
+		opt = opts[0]
+	}
+	m, err := ParseModuleWithOpts("", input, opt)
+	if err != nil {
+		t.Errorf("Error on test \"%s\": parse error on %s: %s", msg, input, err)
+		return
+	}
+
+	if len(m.Rules) != 1 {
+		t.Errorf("Error on test \"%s\": only 1 rule expected, %d parsed", msg, len(m.Rules))
+		return
```

Comment:
Nit: Let's change the error format to something like 

```go
t.Fatalf("Expected 1 rule but got %v", len(m.Rules))
```

Same for other places where were raise an error.